patternfly/patternfly-elements
# Add link to issues in all package.json files

## Added a link to existing issues/bugs in each package.json file
### What has changed and why
Summarize files edited as part of this MR along with a brief description of what was changed/why.

* Each package.json file in the elements directory was updated to include a link to the project's existing issues list.

`"bugs": {
    "url" : "https://github.com/patternfly/patternfly-elements/issues"
  }`

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Run `npm bugs` in the terminal from the project directory. Your default browser should open to the list of current issues in GitHub (https://github.com/patternfly/patternfly-elements/issues).

Your repository infrastructure updates should work for at least:

### Ready-for-merge Checklist
